internal/action/main_test.go

```diff
@@ -0,0 +1,291 @@
+package action
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/hashicorp/go-tfe"
+	"github.com/stretchr/testify/assert"
+)
+
+var workspacePrefix string = "action-test"
+
+func NewTestRunConfig() *RunConfig {
+	return &RunConfig{
+		Token:                  os.Getenv("tf_token"),
+		Organization:           os.Getenv("tf_organization"),
+		Host:                   "app.terraform.io",
+		Name:                   fmt.Sprintf("%s-%d", workspacePrefix, time.Now().Unix()),
+		Import:                 true,
+		Apply:                  true,
+		TFEProviderVersion:     "0.25.3",
+		RunnerTerraformVersion: "1.0.5",
+		TerraformVersion:       "1.0.5",
+	}
+}
+
+func RemoveTestWorkspaces(ctx context.Context, client *tfe.Client, organization string, prefix string) error {
```

Comment:
This can simplify callers by switching to something like `removeTestWorkspaceFunc(t *testing.T, client *tfe.Client) func()`

Then:

```go
t.Cleanup(removeTestWorkspaceFunc(t, client))
```